test/metabase/api/slack_test.clj

```diff
@@ -11,7 +11,8 @@
              `slack-token` setting is cleared"
       (with-redefs [slack/valid-token? (constantly true)
                     slack/channel-exists? (constantly true)
-                    slack/refresh-channels-and-usernames! (fn ([]) ([_]))]
+                    slack/refresh-channels-and-usernames! (constantly nil)
+                    slack/refresh-channels-and-usernames-when-needed! (constantly nil)]
```

Comment:
This tests was failing if run it twice, so adding this to make sure we will not running into it again.